end-to-end/cypress/integration/file-based-routes.spec.js

```diff
@@ -24,7 +24,8 @@ describe('file-based-routes', () => {
       // TODO: When we link to it from the menu, follow the link instead.
       cy.visit('/publication');
       cy.contains('Blue B. Lake');
-      cy.contains('An atlas of healthy and injured cell states').click();
+      // be.visible is a hack: https://github.com/cypress-io/cypress/issues/695
+      cy.contains('An atlas of healthy and injured cell states').should('be.visible').click();
```

Comment:
Was this possibly considered 'not visible' before the hack because the test string provided isn't an exact match of what's on the page?

Reply:
You mean it's just a substring? I don't think that's a problem, but let me check...

Reply:
No, even if it's the full string, the failure is the same:
```diff
-      cy.contains('An atlas of healthy and injured cell states').should('be.visible').click();
+      cy.contains('An atlas of healthy and injured cell states and niches in the human kidney').click();
```
Good to confirm, though.